### PR TITLE
Project 4: TSE Code Review

### PR DESCRIPTION
**Pull request by simoxsimo**
# Apple.com Clone

In this project I have done the following:
- Recreating the site's Layout using Flexbox.
- Using gradients on the NavBar.
- Adding images as backgrounds and using _"background-position"_ to move throughout the image.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
#### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Mohammed!

### Nice job! Project looks! :+1: WOW  the readme file is very descriptive!

### But your project still needs changes.

- Kindly recommended checking HTML markup with this validator https://validator.w3.org/#validate_by_upload.



Now it shows errors in your project

errors


- Please check linter, as I can see it doesn`t work now. You can read more here [Set up a code linter.](https://github.com/microverseinc/linters-config/tree/master/css)

Screenshot from 2019-12-11 00-27-43


- Bottom contains four elements but they should be links (to another articles or #) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))
img

- Please fix footer 

In original version footer is under the bottom
footer

In your version of the [Apple website](https://web.archive.org/web/20140301004610/http://www.apple.com/) webpage footer is in wrong place
foot

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+						<img alt="otherImg1" src="images/otherImgs1.jpg">
+					</div>
+					<div>
+						<img alt="otherImg2" src="images/otherImgs2.jpg">
+					</div>
+					<div>
+						<img alt="otherImg3" src="images/otherImgs3.jpg">
+					</div>
+					<div>
+						<img alt="otherImg4" src="images/otherImgs4.jpg">
```
Comment (Anna-Myzukina) [inline index.html:66]: This four elements should be links (to another articles or `#`)

**Comment by simoxsimo** [comment]
Pull Request Closed to fix Stickler config

**Pull request closed without merging**